### DISCUSSION TITLE
Gradle build now uses path-based task names

### DISCUSSION THREAD

**Discussion by anuraaga** [open-telemetry/opentelemetry-java#2354] [Q&A]
#2332 has updated the build to use the folder names in the repository for Gradle task names.

Previously `./gradlew :opentelemetry-api:check` is now `./gradlew :api:all:check`.

The intent of this change is 

- Make commands a bit shorter
- Reduce cognitive load by not having to know the naming scheme that produced tasks previously.

Sorry for the break in muscle memory but hopefully the shorter commands make it worth it :)

Happy hacking!

**Comment by jkwatson** [comment]
can we add something to CONTRIBUTING.md about this?

**Reply by anuraaga** [reply]
I think previously we might have since we were using custom task names. Now that they're the default, I'm not sure what to add - "Don't worry, we use the default task names"? :P

**Reply by jkwatson** [reply]
I just thought I would see how discussions work, mostly.  ;)
